### ISSUE TITLE
Rework chat triggers

### ISSUE THREAD

**Issue by shelltar** [NightmareXIV/NotificationMaster#7]
Is it possible to add location to the triggers? For example, I don't want to be notified of every /say in capital cities, but I'd like it in housing zones.

**Comment by Limiana** [comment]
Yes, I'm planning to completely rework chat triggers, I will get on this as soon as I have time
